Entity/Location.cs

```diff
@@ -0,0 +1,13 @@
+namespace DurgerKing.Entity;
+
+public class Location
+{
+    public Guid Id { get; set; }
```

Comment:
long bolishi kerak
shunda long UserId ham kerak emas. 
One-to-One relationship qilishda oson boladi.v


Eski vidoelarimizda qarab User va DriverLicense qanday qilingaini korib oling